pandas/core/generic.py

```diff
@@ -7527,26 +7528,39 @@ def asfreq(
         fill_value=None,
     ) -> FrameOrSeries:
         """
-        Convert TimeSeries to specified frequency.
-
-        Optionally provide filling method to pad/backfill missing values.
+        Convert time series to specified frequency.
 
         Returns the original data conformed to a new index with the specified
-        frequency. ``resample`` is more appropriate if an operation, such as
-        summarization, is necessary to represent the data at the new frequency.
+        frequency.
+
+        If the index of this {klass} is a :class:`~pandas.PeriodIndex`, the new index
+        is the result of transforming the original index with
+        :meth:`PeriodIndex.asfreq pandas.PeriodIndex.asfreq` (so the original index
```

Comment:
Did you mean to write it twice here?

Reply:
Hmm, I don't see it, what do you mean?

Reply:
```
:meth:`PeriodIndex.asfreq pandas.PeriodIndex.asfreq`
```

Is it correct to have `PeriodIndex.asfreq` twice?

Reply:
Oh I see! That's Sphinx syntax, the first part is the link text (`PeriodIndex.asfreq`) and the second part is the actual link path (`pandas.PeriodIndex.asfreq`). I did this just to omit the leading `pandas.` from the link text. (Here `~` wouldn't do it because it would just show `asfreq`.)

Reply:
I'm probably missing something, but when I build this locally it shows up twice rather than having a link:
![image](https://user-images.githubusercontent.com/33491632/93300610-03219b80-f7ef-11ea-8db2-89c757c222a8.png)

Reply:
Should it be something like
```suggestion
        :meth:`PeriodIndex.asfreq <pandas.PeriodIndex.asfreq>` (so the original index
```
?

Reply:
Ah, you're totally correct! For some reason I interpreted the angle brackets as not part of the syntax itself—and I didn't notice that when checking the output.